src/sentry_plugins/bitbucket/testutils.py

```diff
@@ -0,0 +1,295 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+
+# we keep this as a raw string as order matters for hmac signing
```

Comment:
you can keep this as a raw string, but since we're not using hmac signing for bitbucket, wanna delete this comment?